<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompoundWidget</class>
 <widget class="QWidget" name="CompoundWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>合成表达式</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择操作符与两个前缀表达式</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8,8">
     <item>
      <widget class="QComboBox" name="opComboBox">
       <item>
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>*</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>/</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>^</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="OprandBox1"/>
     </item>
     <item>
      <widget class="QComboBox" name="OprandBox2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>128</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="readButton">
       <property name="text">
        <string>读入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
